provision/kubernetes/node.go

```diff
@@ -74,10 +75,25 @@ func (n *kubernetesNodeWrapper) ExtraData() map[string]string {
 }
 
 func (n *kubernetesNodeWrapper) Units() ([]provision.Unit, error) {
-	pods, err := appPodsFromNode(n.cluster, n.node.Name)
+	pools, err := pool.ListAllPools()
```

Comment:
Do we need to list all pools here since the node belong to a single pool?

Reply:
We need the namespace to list pods with a specific selector in https://github.com/ggarnier/tsuru/blob/8a066b8e3d34f6f5a6836c3962bbd42f04fa7989/provision/kubernetes/helpers.go#L413.

But with this alternative you just found out to list pods from all namespaces could replace this.